docs/documentation/maintenance-coordinator.md

```diff
@@ -0,0 +1,91 @@
+Pull requests on the public interface must be accepted by the JF. The coordinator gives a 
+recommendation to the JF on whether to accept or decline the PR. The decision of the JF may 
+be implicit if no objections to the recommendation of the coordinator is made. If no agreement 
+is achieved in the JF, the Technical Board will decide upon the request.
+
+Final implementations without further changes on the interface do not need 
+formal approval by the JF. The merge of the implementation is performed 
+by the coordinator or the coordinator may assign somebody to do so.
+
+Note that the general process for feature requests must be respected. However,
+this process is currently under review. The respective document will be linked as soon
+as available.
+
+<a name="issue-management"></a>
+## Issue Management
+Everybody is invited to make proposals on how to tackle any issue by proposing 
+a respective PR. Issues of the component must be reported as bugs. The coordinators 
+are responsible to assign the developer in charge on solving the bug. Due to the focus 
+on code quality, a low amount of bugs should be expected.
+
+<a name="scenarios"></a>
+## Scenarios
+This maintainance model is suited for components that have the potential 
+to grow too large to be handled by one single developer and therefore highly 
+benefit from contributions among different developers and even service providers. 
+It is further important that the component is of modular structure with different 
+parts following a similar scheme. It is especially suited, if some component is 
+of a critical importance for many other components, since it is designed to allow 
+a collaborative development of the vision for such a key aspect.
+
+<a name="expectations"></a>
+## What can be expected of a coordinator?
+* The coordinator MUST moderate the discussion on finding a vision on 
+the development of the component.
+* The coordinator MUST give recommendations to the JF whether to accept 
+or decline changes.
+* The coordinator MUST accept decisions of the Technical Board on change 
+requests in case of disagreement on the JF.
+* The coordinator MUST review final implementation of some accepted 
+interface change or organize some substitute to perform the review.
+* The coordinator MAY ask for funding to perform the review.
+* The coordinator MAY ask to split some interface change or implementation 
+into multiple pieces to make the change easier to understand.
+* The coordinator MUST ensure, that the documentation of the component is 
+up to date. The coordinator MAY ask to update the documentation as a condition 
+for accepting some change request.
+* The coordinator MUST ensure, that the automated unit tests are kept up 
+to date. The coordinator MAY ask other developers to write and update those 
+tests in their own development.
+* The coordinator MUST assign unassigned issues to the responsible developer.
```

Comment:
Good point. This is related to the problem in the component maintainer model. What if a maintainer leaves without any body willing to fill it's shoes? Also, note that this is a problem for currently unmaintained parts of the code (parts folders without anybody entered in the maintainance.json). If an issue is not assignable to anyone it remains on the coordinators desk.  Further steps are up to the coordinator (look for funding, communicate the issue to the other devs and check for possible replacement).

Reply:
Maybe a sentence would help that the coordinator task is to find appropriate candidates if the candidates are gone. Also a sentence that the coordinator can also address bottlenecks and flaws to the JF or TB to look for help? I don't know if this is common sense in this project, but for newcomers this could be a bit confusing. What do you think?

Reply:
I like your reasoning. Any concrete proposal for such a sentence we can discuss in the JF?

Reply:
As a start I suggest to add a sentence like "The coordinator is NOT responsible to fix issues of other developers. Coordinators SHOULD mark unmaintained code and inform the TB."